<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keyboard</class>
 <widget class="QWidget" name="Keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_x">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_y">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_v">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_t">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_i">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_c">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>230</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_f">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>65</y>
     <width>40</width>
     <height>105</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_q">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_g">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_l">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_k">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_j">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_a">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_h">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_r">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BIG">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>175</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_o">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_z">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>230</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_u">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_m">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>230</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>13</y>
     <width>480</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border: white;           
background: transparent; 
      

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>999</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_s">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_d">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>175</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_w">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_b">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>230</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_e">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_n">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>65</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_p">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
</string>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
